cmd/terramate/e2etests/cloud/run_cloud_deployment_test.go

```diff
@@ -364,32 +371,12 @@ func TestCLIRunWithCloudSyncDeployment(t *testing.T) {
 				addr := startFakeTMCServer(t, cloudData)
 
 				s := sandbox.New(t)
-				var genIdsLayout []string
-				ids := []string{}
-				if !tc.skipIDGen {
-					for _, layout := range tc.layout {
-						if layout[0] == 's' {
-							if strings.Contains(layout, "id=") {
-								t.Fatalf("testcases should not contain stack IDs but found %s", layout)
-							}
-							id := strings.ToLower(strings.Replace(layout[2:]+"-id-"+t.Name(), "/", "-", -1))
-							if len(id) > 64 {
-								id = id[:64]
-							}
-							ids = append(ids, id)
-							layout += ":id=" + id
-						}
-						genIdsLayout = append(genIdsLayout, layout)
-					}
-				} else {
-					genIdsLayout = tc.layout
-				}
```

Comment:
this was created when no special handling for `stack.id` was required, then generating them made it easier to test but too risky to keep now that case insensitivity is handled.